common/core/overlaylayer.cpp

```diff
@@ -416,8 +416,8 @@ void OverlayLayer::ValidatePreviousFrameState(OverlayLayer* rhs,
     if (!rhs->imported_buffer_.get()) {
       state_ |= kNeedsReValidation;
       return;
-    } else if (buffer->GetFormat() !=
-               rhs->imported_buffer_->buffer_->GetFormat()) {
+    } else if (buffer && (buffer->GetFormat() !=
+               rhs->imported_buffer_->buffer_->GetFormat())) {
```

Comment:
@jennycao There is still a travisCI failure due to a clang-format error. I ran the instructions found here:[Wiki:Conformance](https://github.com/intel/IA-Hardware-Composer/wiki/Conformance#clang-format) and got the result below. Once this change is made, it should be good to merge.

user@pc:~/projects/gma/hwc$ git diff | clang-format-diff-3.9 -p 1 -style=file
--- common/core/overlaylayer.cpp	(before formatting)
+++ common/core/overlaylayer.cpp	(after formatting)
@@ -417,7 +417,7 @@
               state_ |= kNeedsReValidation;
               return;
             } else if (buffer && (buffer->GetFormat() !=
-                          rhs->imported_buffer_->buffer_->GetFormat())) {
+                                          rhs->imported_buffer_->buffer_->GetFormat())) {
               state_ |= kNeedsReValidation;
               return;
             }

In addition, I updated the wiki to make the instructions more clear. Let me know if you have any questions or have anything I can help you with.

Reply:
@stripes416 changed and pass the clang-format check Thanks !